packages/deployment/app/config.yml

```diff
@@ -83,6 +83,9 @@ default:
     provider: CUMULUS
     clientId: CUMULUS
     password: '{{CMR_PASSWORD}}'
+    cmrEnvironment: UAT
```

Comment:
There should be entries in the deployment docs for this. https://nasa.github.io/cumulus/docs/deployment/config_descriptions

Reply:
added